code/modules/clothing/modular_armor/modular.dm

```diff
@@ -406,3 +406,79 @@
 	. = ..()
 	. += "<br><br />This is a piece of modular armor, It can equip different attachments.<br />"
 	. += "<br>It currently has [attachments_by_slot[ATTACHMENT_SLOT_HEAD_MODULE] ? attachments_by_slot[ATTACHMENT_SLOT_HEAD_MODULE] : "nothing"] installed."
+
+/** Colorable masks */
+/obj/item/clothing/mask/modular
```

Comment:
Is there a particular reason this isn't a child of gasmasks? It seems like theres a lot of stuff thats just copy paste from there.